src/form/ivr.jl

```diff
@@ -112,13 +112,13 @@ function variable_mc_bus_voltage(pm::_PM.AbstractIVRModel; nw=pm.cnw, bounded::B
 
     ncnd = length(conductor_ids(pm))
     theta = [_wrap_to_pi(2 * pi / ncnd * (1-c)) for c in 1:ncnd]
-    vm = 1
+    vm = haskey(ref(pm, nw, :bus, id), "vm_start") ? busref["vm_start"] : fill(1.0, ncnd)
     for id in ids(pm, nw, :bus)
         busref = ref(pm, nw, :bus, id)
         if !haskey(busref, "va_start")
             for c in 1:ncnd
-                vr = vm*cos(theta[c])
-                vi = vm*sin(theta[c])
+                vr = vm.*cos(theta[c])
+                vi = vm.*sin(theta[c])
```

Comment:
I think you need to put `[c]` on `vm`, otherwise when you add the test it will error out because you're trying to set the start value as an array